### PR TITLE
Giashard recipe

### PR DESCRIPTION
**Pull request by cgr71ii**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by cgr71ii** [comment]
@conda-forge/staged-recipes, ready for review!

File: recipes/giashard/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/paracrawl/giashard
+  git_rev: {{ commit }}
+  sha1: {{ sha1value }}
+
```
[the remark this answers is not in the document]
Comment (ocefpaf) [reply recipes/giashard/meta.yaml:14]: We discourage this but... If upstream is difficult and the software is really important we did open a few exceptions in the past. With that said we ned first:

- open an issue or some communication with upstream asking for a proper release
- if that fails, make a case that this sotware is essential for reasons xyz

[the remark this answers is not in the document]
Comment (cgr71ii) [reply recipes/giashard/meta.yaml:14]: Hi! Thank you for the suggested changes! I'm working on it and I hope to have this fixed early.

[the remark this answers is not in the document]
Comment (cgr71ii) [reply recipes/giashard/meta.yaml]: Solved! Now a release is being used instead of a specific commit.

**Review by carterbox — CHANGES_REQUESTED** [review]
This package has external go package dependencies, the licenses for these packages must be included. See this discussion for help.

https://github.com/conda-forge/conda-forge.github.io/issues/1052

**Pull request closed without merging**